styles.css

```diff
@@ -619,6 +703,144 @@ button:active {
     background-color: #fff;
     width: 100%;
   }
+
+  .modal-desktop-overlay {
+    width: 100%;
+    background-color: #c1c7d0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .desktop-popup-container {
+    display: flex;
+    padding: 10px;
+    background-color: white;
+    flex-direction: column;
+    flex-wrap: wrap;
+    z-index: 2;
+  }
+
+  .desktop-project-description {
+    margin: 0 4%;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+  }
+
+  .cancel-icon-desktop {
+    position: absolute;
+    width: 30px;
+    top: 20px;
+    right: 60px;
+    background-color: white;
+  }
+
+  .heading-button-container {
+    display: flex;
+    margin: 5% 4% 0;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+  }
+
+  .popup-technologies {
+    margin: 0 4%;
+    justify-content: flex-start;
+    background-color: white;
+  }
+
+  .popupbutton-div {
+    text-align: center;
+  }
+}
+
+.popup-section {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  position: absolute;
+  background-color: white;
+  transition: all 0.4s, top 0.4s;
+}
```

Comment:
- [ ] Kindly make sure your popup container is fixed-positioned. On your website, the pop-up is being opened but we can still scroll to see other parts of the page, however when the pop-up is opened we must not be able to see other parts of the page, we must only see the pop-up.

- [ ] Kindly make sure to add the background color on your popup container in to look like the provided Figma design. 

  - ### YOU POP-UP

   ![chrome-capture-2022-11-23 (4)](https://user-images.githubusercontent.com/87197412/209373534-7ce04080-93c4-4249-bf12-af3ae2f28fc2.gif)

   ![chrome-capture-2022-11-23 (7)](https://user-images.githubusercontent.com/87197412/209375286-f97a5890-3542-42d6-8204-752bfac0c3f2.gif)


  - ### FIGMA DESIGN

   ![image](https://user-images.githubusercontent.com/87197412/209374759-e795e162-98a6-403d-ae7a-bcba192d6061.png)


  ![image](https://user-images.githubusercontent.com/87197412/209373439-c8b80a21-63ad-4d19-be14-bcc24d0ae35b.png)